tests/test_basic.py

```diff
@@ -2351,6 +2351,18 @@ def test_edge_collapse_is_valid(self):
         #val = igl.edge_collapse_is_valid(0, self.f2, e, emap, ef, ei)
         #print(val)
 
+    def test_flip_edge(self):
+        e, ue, emap, ue2e = igl.unique_edge_map(self.f1)
+        f, e, ue, emap, ue2e = igl.flip_edge(self.f1, e, ue, emap, ue2e, 1)
+
+        self.assertTrue(e.shape[1] == ue.shape[1] == 2)
+        self.assertTrue(emap.shape[0] == self.f1.shape[0] * 3)
+        self.assertTrue(np.min(e) >= 0 and np.max(e) < self.v2.shape[0])
+        self.assertTrue(e.flags.c_contiguous)
+        self.assertTrue(emap.flags.c_contiguous)
+        self.assertTrue(ue.flags.c_contiguous)
+        self.assertTrue(e.dtype == emap.dtype == self.f1.dtype)
```

Comment:
can you add assertions for the dtypes and sizes of all returns (dtypes are missing for all and sizes for f)